buildingmotif/label_parsing/docs/usage.py

```diff
@@ -0,0 +1,170 @@
+usage_markdown = r"""
```

Comment:
We should add some documentation on how this `usage_markdown` was created. Did I send you code for this, or is it something you developed?